### PR TITLE
WT3 costing municipal drinking

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
Included in Santa Barabara case study- this is a first attempt at applying the new costing framework to a simple WT3 model.

## Changes proposed in this PR:
- edit yaml
- add test

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by luohezhiming — DISMISSED** [review]
LGTM

File: watertap/data/techno_economic/municipal_drinking.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-    function f(flow_vol_in): cap_basis_par*(flow_vol_in)^cap_exp_par
-    units: $
+  capital_cost:
+    basis: flow_vol
+    cost_factor: TPEC
+    reference_state:
+      value: 1
+      units: Mgallons/day
+    capital_a_parameter:
+      value: 0.0403
```
Comment (bknueven) [inline watertap/data/techno_economic/municipal_drinking.yaml]:
```suggestion
      value: 0.0403e6
```
since the prior unit was `M$`?

**Comment by andrewlee94** [comment]
Besides @bknueven's comment, this looks good.

**Review by bknueven — APPROVED** [review]

**Review by andrewlee94 — APPROVED** [review]
Looks good to me.

**Pull request merged**